src/wymeditor/core.js

```diff
@@ -541,251 +831,39 @@ jQuery.extend(WYMeditor, {
-                '</ul>' +
-            '</div>',
-
-        containersItemHtml: String() +
-            '<li class="' + WYMeditor.CONTAINER_CLASS + '">' +
-                '<a href="#" name="' + WYMeditor.CONTAINER_NAME + '">' +
-                    WYMeditor.CONTAINER_TITLE +
-                '</a>' +
-            '</li>',
-
-        containersItems: [
-            {'name': 'P', 'title': 'Paragraph', 'css': 'wym_containers_p'},
-            {'name': 'H1', 'title': 'Heading_1', 'css': 'wym_containers_h1'},
-            {'name': 'H2', 'title': 'Heading_2', 'css': 'wym_containers_h2'},
-            {'name': 'H3', 'title': 'Heading_3', 'css': 'wym_containers_h3'},
-            {'name': 'H4', 'title': 'Heading_4', 'css': 'wym_containers_h4'},
-            {'name': 'H5', 'title': 'Heading_5', 'css': 'wym_containers_h5'},
-            {'name': 'H6', 'title': 'Heading_6', 'css': 'wym_containers_h6'},
-            {'name': 'PRE', 'title': 'Preformatted', 'css': 'wym_containers_pre'},
-            {'name': 'BLOCKQUOTE', 'title': 'Blockquote',
-                'css': 'wym_containers_blockquote'},
-            {'name': 'TH', 'title': 'Table_Header', 'css': 'wym_containers_th'}
-        ],
-
-        classesHtml: String() +
-            '<div class="wym_classes wym_section">' +
-                '<h2>{Classes}</h2>' +
-                '<ul>' +
-                    WYMeditor.CLASSES_ITEMS +
-                '</ul>' +
-            '</div>',
-
-        classesItemHtml: String() +
-            '<li class="wym_classes_' + WYMeditor.CLASS_NAME + '">' +
-                '<a href="#" name="' + WYMeditor.CLASS_NAME + '">' +
-                    WYMeditor.CLASS_TITLE +
-                '</a>' +
-            '</li>',
-
-        classesItems:      [],
-        statusHtml: String() +
-            '<div class="wym_status wym_section">' +
-                '<h2>{Status}</h2>' +
-            '</div>',
-
-        htmlHtml: String() +
-            '<div class="wym_html wym_section">' +
-                '<h2>{Source_Code}</h2>' +
-                '<textarea class="wym_html_val"></textarea>' +
-            '</div>',
-
-        boxSelector:        ".wym_box",
-        toolsSelector:      ".wym_tools",
-        toolsListSelector:  " ul",
-        containersSelector: ".wym_containers",
-        classesSelector:    ".wym_classes",
-        htmlSelector:       ".wym_html",
-        iframeSelector:     ".wym_iframe iframe",
-        iframeBodySelector: ".wym_iframe",
-        statusSelector:     ".wym_status",
-        toolSelector:       ".wym_tools a",
-        containerSelector:  ".wym_containers a",
-        classSelector:      ".wym_classes a",
-        htmlValSelector:    ".wym_html_val",
-
-        updateSelector:    ".wymupdate",
-        updateEvent:       "click",
-
-        stringDelimiterLeft:  "{",
-        stringDelimiterRight: "}",
-
-        preInit: null,
-        preBind: null,
-        postInit: null
-
-    }, options);
```

Comment:
We do have some kind of multi–file system. How about pulling this out into its own file?